Makefile

```diff
@@ -76,7 +76,7 @@ serve:
 	node server.js
 
 test:
-	./node_modules/.bin/jasmine-node test/katex-spec.js
+	./node_modules/.bin/jasmine-node test/*-spec.js
```

Comment:
Switching to mocha sounds fine to me! It might take some work to convert our matchers, but sounds like it'll be worth it.